fe/src/project/AspNetCore/areas/News/components/NewsCatalog/NewsCatalog.js

```diff
@@ -0,0 +1,156 @@
+import React, { useContext, useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import LazyLoad from "react-lazyload";
+import LazyLoadConstants from "../../../../../../shared/constants/LazyLoadConstants";
+import ResponsiveImage from "../../../../../../shared/components/Picture/ResponsiveImage";
+import QueryStringSerializer from "../../../../../../shared/helpers/serializers/QueryStringSerializer";
+import NavigationHelper from "../../../../../../shared/helpers/globals/NavigationHelper";
+import { Context } from "../../../../../../shared/stores/Store";
+import { CalendarToday } from "@material-ui/icons";
+import { Hash } from "react-feather"
+import moment from "moment";
+
+const NewsCatalog = (props) => {
+    const [state, dispatch] = useContext(Context);
+    const [items, setItems] = useState(props.pagedResults.data ? props.pagedResults.data : null);
+    const [total] = useState(props.pagedResults.total ? props.pagedResults.total : null)
+    const [hasMore, setHasMore] = useState(true);
+
+    let pageIndex = 2;
+    const handleLoadNews = async () => {
+        if (total < 10) return;
+
+        dispatch({ type: "SET_IS_LOADING", payload: true });
+        const requestOptions = {
+            method: "GET",
+            headers: { "Content-Type": "application/json", "X-Requested-With": "XMLHttpRequest" }
+        }
+
+        const queryStrings = {
+            itemsPerPage: 9, 
```

Comment:
**suggestion:** avoid using magic strings in code. Inject 10 via props from backend. In the backend put this value to Constants and give it a descriptive and meaningful name.